quinn-proto/src/endpoint.rs

```diff
@@ -592,10 +583,22 @@ impl Endpoint {
         }
     }
 
+    /// Reject this incoming connection attempt
+    pub fn reject(&mut self, incoming: IncomingConnection, buf: &mut BytesMut) -> Transmit {
```

Comment:
bikeshed: should we name this "refuse", to match the RFC terminology and the error code name?

Reply:
I can if you want, but there's also a lot of existing places in the repository where the word "reject" is used, such as in `reject_new_connections`. Thoughts?

Reply:
Hmm, good point. No strong opinion; we can revisit in a quick follow-up if we want.